pkg/hyper/helper.go

```diff
@@ -187,13 +187,18 @@ func buildLabelsWithAnnotations(labels, annotations map[string]string) map[strin
 		return labels
 	}
 
+	newLables := make(map[string]string)
+	if labels != nil {
+		newLables = labels
```

Comment:
Yes, will do. `make(map[string]string)` will take more resource  than `newLables = labels`. So `newLables = labels` is better. Am I right?